### ISSUE TITLE
Suboptimal performance

### ISSUE THREAD

**Issue by vmcrash** [vlang/ui#226]
**V version:** V 0.1.29 4d2c8a4
**UI version:** 1e0c0e6
**OS:** Win10

**What did you do?**
`examples> v run users.v` or `examples> users.exe`

**What did you expect to see?**
I expect to see the tiny UI immediately (finally, it is native code).
 
**What did you see instead?**
I first see the *V UI Demo* window occur, after ~2s I see the content with some massive flickering. Even Java applications launch faster.

**Comment by medvednikov** [comment]
`v run` builds programs before running them. Use `v users.v` then `users.exe`.

Building is slow because modules are not cached right now, and entire ui module is rebuilt every time.

**Comment by medvednikov** [comment]
flickering shouldn't happen, need to investigate

**Comment by vmcrash** [comment]
Please reread the issue. I was **not** writing about the compilation time, but the time between showing the window outline and its content. The problem is reproducible even when launching `users.exe` (or any other one from the `examples` directory) directly.

**Comment by serkonda7** [comment]
Do you experience the same delay even if you launch a exe build with `v - prod ... `?

**Comment by medvednikov** [comment]
Sounds like a driver issue. OpenGL is used on Windows instead of D3D at the moment.

**Comment by vmcrash** [comment]
> 
> 
> Do you experience the same delay even if you launch a exe build with `v -prod ... `?

Yes, same delay. Do you need any further information from my system?

**Comment by medvednikov** [comment]
Can you record a demo? That'd be super useful.

**Comment by vmcrash** [comment]
![users](https://user-images.githubusercontent.com/853847/92417022-72005400-f160-11ea-8ecd-c16fb004c289.gif)

**Comment by medvednikov** [comment]
Thanks. So it's not flickering, it's just loading something on init. Could be a Windows antivirus scan thing?

This doesn't happen on other systems. But I'll investigate with a windows vm.

**Comment by hubyhuby** [comment]
On linux Ubuntu 18.04 it is super fast.
